inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -26,22 +26,22 @@ class SourceParameters:
 
 @dataclass(kw_only=True, frozen=True)
 class ApplicationAddSourceParameters:
-    file_name: str
-    sources: list[str] = field(default_factory=lambda: [])
+    source_list_file_name: str
```

Comment:
Ensure that the renaming of the `file_name` parameter to `source_list_file_name` is reflected everywhere in the codebase and does not break any existing functionality. [important]